seumich/static/seumich/images/external-link.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
```

Comment:
This SVG is not needed, already included as fa-external-link

Reply:
Nice! I have updated the patch.